rflx/expression.py

```diff
@@ -1710,8 +1710,21 @@ def _simplified(self, relation_operator: Callable[[Expr, Expr], bool]) -> Expr:
         }
         if (relation_operator, left, right) in mapping:
             return mapping[(relation_operator, left, right)]
-        if isinstance(left, Number) and isinstance(right, Number):
-            return TRUE if relation_operator(left, right) else FALSE
+        if isinstance(left, (Number, Aggregate)) and isinstance(right, (Number, Aggregate)):
+            if isinstance(left, Number) and isinstance(right, Number):
+                return TRUE if relation_operator(left, right) else FALSE
+            if isinstance(left, Aggregate) and isinstance(right, Aggregate):
+                return TRUE if left.elements == right.elements else FALSE
+            aggregate = left if isinstance(left, Aggregate) else right
+            number = left if isinstance(left, Number) else right
+            assert isinstance(aggregate, Aggregate)
+            assert isinstance(number, Number)
+            if relation_operator is operator.eq and number in aggregate.elements:
+                return TRUE
+            if relation_operator is operator.ne and number not in aggregate.elements:
+                return TRUE
```

Comment:
`=` and `/=` are only for checking equality or inequality. These operators should not be interpreted as `in` or `not in`.

Reply:
Isn't the expression `Fld_A = [16#01#, 16#02#]` checking if the left side is qual to any element of the sequence on the right side? If so, it is not clear to my why the "(not) in"  operator should not be used, I could rewrite it as a loop and check for equality, but in my mind this would be the same thing.

Reply:
`=` is an equality operator, which checks that both sides are the same. `=` is _not_ an `in` operator, which checks if a number is part of an aggregate.

The condition `Fld_A = [16#01#, 16#02#]` must get true only if `Fld_A` is an opaque field with a size of two bytes (or a sequence of two integers), where the first byte of the field is `1` and the second byte is `2`. If `Fld_A` is not an opaque field (or a sequence of integers), the specification should be rejected during model verification.

Reply:
That was a misunderstanding on my side. Somehow I was firmly convinced that this should be a way to express `in`. My bad. As a consequence, I needed to change the implementation of this feature fundamentally in 9ab17057